post-types/templates/single-tabular.php

```diff
@@ -8,16 +8,15 @@
 		$valid_config = true;
 
 		$dataset_type = get_post_meta($post->ID, '_attributes_dataset_type', true);
-		$column_list = (odm_language_manager()->get_current_language() != "en") ? get_post_meta($post->ID, '_attributes_column_list_localization', true) : get_post_meta($post->ID, '_attributes_column_list', true);
-		$values_mapping = (odm_language_manager()->get_current_language() != "en") ? get_post_meta($post->ID, '_attributes_values_mapping_localization', true) : get_post_meta($post->ID, '_attributes_values_mapping', true);
+		$column_list = get_post_meta($post->ID, '_attributes_column_list', true);
```

Comment:
@Huyeng your PR removes some code changes that I did which are very important:
- adding multilingual support on the config